src/app/data/modelDTO/animal-dto.ts

```diff
@@ -0,0 +1,60 @@
+export class AnimalDto {
+    private age: string;
+    private birthDay: string;
+    private bloodType: string;
+    private name: string;
+    private weight: string;
+
+    constructor(age = '', birthDay = '', bloodType = '', name = '', weight = '') {
```

Comment:
Use the constructor or builder pattern - don't use both because you wil have duplicate code